08-interop/README.md

```diff
@@ -281,3 +281,188 @@ guide on Zephyr's documentation.
    dst_l2addr: 32:A9:FA:65:10:6B:11:14
    ~~ PKT    -  4 snips, total size:  85 byte
    ```
+   
+Task #12 - GNRC Border Router with WiFi uplink to the Internet
+========================
+### Description
+
+This test will ensure RIOT can connect to a real IPv6 enabled WiFi network and
+share the uplink connection with constrained nodes.
+
+An esp* board with the `gnrc_border_router` connects to your WiFi.
+The module `sock_dns` is used to resolve domain names.
+
+Two network interfaces are configured:
+
+ - a 802.11 WiFi interface used as uplink
+ - a proprietary `esp_now` interface as 6LoWPAN downlink
+ 
+ Materials
+---------
+
+You'll need:
+
+ - a pair of esp8266/esp32 boards
+ - 2.4 GHz WiFi network with an IPv6 uplink
+ - to make sure prefix delegation (IA_PD) is enabled in your router's DHCPv6 server
+   (On the popular Fritz!Box line of routers this can be enabled in the
+   Network -> Network Settings -> IPv6 Addresses menu)
```

Comment:
Maybe also add a link at least for OpenWrt: https://openwrt.org/docs/guide-user/network/ipv6/start

... For some FOSS cross-pollination ;-)

Reply:
Looks like it's enabled by default

Reply:
I would add the link regardless, just to be sure.